src/main/groovy/com/github/jrubygradle/JRubyPrepareJars.groovy

```diff
@@ -0,0 +1,55 @@
+package com.github.jrubygradle
+
+import com.github.jrubygradle.internal.JRubyExecUtils
+import org.gradle.api.DefaultTask
+import org.gradle.api.Project
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+
+/**
+ * @author Schalk W. Cronjé
+ * @author R Tyler Croy
+ * @author Christian Meier
+ */
+class JRubyPrepareJars  extends DefaultTask {
+
+    /** Target directory for GEMs. Extracted GEMs should end up in {@code outputDir + "/jars"}
+     */
+    @OutputDirectory
+    File outputDir
```

Comment:
Change this block to rather read 
```groovy
@OutputDirectory
File getOutputDir() { project.file(this.outputDir) }

void outputDir(Object file) {this.outputDir = file}

@PackageScope
Object outputDir
```

This allows for lazy evaluation to be delayed to the point of execution.